st2common/st2common/services/keyvalues.py

```diff
@@ -256,3 +260,34 @@ def get_key_reference(scope, name, user=None):
         raise InvalidScopeException(
             'Scope "%s" is not valid. Allowed scopes are %s.' % (scope, ALLOWED_SCOPES)
         )
+
+
+def get_uids(user):
```

Comment:
```suggestion
def get_uid(user):
```
Why is this `uids`?

Reply:
Because it is returning the specific uid that RBAC assign to a resource. (i.e. key_value_pair:st2kv.system:key1).  It's plural because the function is trying to return all (multiple) resources that have been explicitly granted for a user.